dialogy/plugins/text/classification/xlmr.py

```diff
@@ -44,26 +48,15 @@ def __init__(
         state_column: str = const.STATE,
         lang_column: str = const.LANG,
         nls_label_column: str = const.NLS_LABEL,
-        args_map: Optional[Dict[str, Any]] = None,
```

Comment:
Why would this go away?
I would still expect the runtime parameters of XLMR plugin to be passed in this `args_map` format:
```
train:
   param1: ..
   
eval:
   param2: ..
   
production:
   param1: ..
```
The plugin can pickup the correct set of params according to the passed `purpose` at runtime.

Reply:
reverted